docs/book/starter-guide/pipelines/fetching-pipelines.md

```diff
@@ -155,10 +155,33 @@ zenml pipeline runs list --created "gt:2021-01-01 00:00:00"
 ```
 
 Please [refer to our
-documentation](https://apidocs.zenml.io/0.38.0/cli/#zenml.cli--filtering-cli-output-when-listing)
+documentation](https://apidocs.zenml.io/latest/cli/#zenml.cli--filtering-cli-output-when-listing)
 for full details on the available flags.
 </details>
 
+## Querying the Client
```

Comment:
Should be h3 since runs is h2

```suggestion
### Querying the Client
```

Reply:
Actually now that I look at the page, how about we keep it a h2 header but move it to the bottom of the page after "Code Example"?